test/service/test_repository_proxy.py

```diff
@@ -125,7 +129,10 @@ class TestTDRRepositoryProxy(RepositoryPluginTestCase):
                        '_http_client',
                        AuthorizedHttp(MagicMock(),
                                       urllib3.PoolManager(ca_certs=certifi.where())))
-    def test_repository_files_proxy(self):
+    def test_repository_files_proxy(self,
+                                    mock_get_cached_sources,
+                                    ):
```

Comment:
```suggestion
    def test_repository_files_proxy(self, mock_get_cached_sources):
```
The line wrapping and trailing comma seems unnecessary . The only other method in this file with this formatting is `test_list_sources` where it also seems out of place. Was it styled this way for the diff?

Reply:
I think this was a result of changes I made and then reverted.

Reply:
In the cast of `test_list_sources` it is for the diff.